polling_stations/apps/data_finder/helpers/baked_data_helper.py

```diff
@@ -80,8 +113,9 @@ def get_ballot_list(self, postcode: Postcode, uprn: Optional[str] = None):
             # In theory this shouldn't happen
             # every outcode should exists as a parquet file
             message = f"Expected file {parquet_filepath} not found"
-            logger.error(message)
-            capture_message(message, level="error")
+            scope = get_current_scope()
+            scope.fingerprint = ["parquet:expected_parquet_file_not_found"]
+            logging.error(message)
```

Comment:
Setting a "fingerprint" for each class of error helps Sentry to group related errors as different "events" for the same "error".
When I hooked this up to a real Sentry DNS, I also realised here that I don't need to be calling both `logging.error` and `capture_message` for these. Sentry's log handler will pick up error logs.